crankshaft-console/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
To get lints to pass, we need a short `//!` comment describing the purpose of the crate; it doesn't have to be very long right now since this is just a stub crate.